Packs/rasterize/Integrations/rasterize/rasterize_test.py

```diff
@@ -364,3 +375,183 @@ def test_get_output_filenames():
     assert get_list_item(file_names, 2, "FOO.png") == 'foo_03.png'
     assert get_list_item(file_names, 3, "FOO.png") == 'FOO.png'
     assert get_list_item(file_names, 4, "FOO.png") == 'FOO.png'
+
+
+def test_info_file_not_exist_then_create_new_browser(mocker):
+    from rasterize import chrome_manager
+
+    instance_id = "instance_id"
+    chrome_options = "chrome_options"
+
+    mock_context = {
+        'context': {
+            'IntegrationInstanceID': instance_id
+        }
+    }
+
+    params = {
+        'chrome_options': chrome_options
+    }
+
+    mocker.patch.object(demisto, 'callingContext', mock_context)
+    mocker.patch.object(demisto, 'params', return_value=params)
+    mocker.patch.object(rasterize, 'read_file', return_value=None)
+    mocker.patch.object(rasterize, 'get_chrome_instances_contents_dictionaries', return_value=[{}, {}, {}])
+    mocker.patch.object(rasterize, 'generate_new_chrome_instance', return_value=["browser_object", "chrome_port"])
+    browser, chrome_port = chrome_manager()
+
+    assert browser == "browser_object"
```

Comment:
Altough I understand all those test (this one and the next 3) are testing separate cases. the assertions are the same. I think we can add assertions to validate we are at the correct case (you can insert the mock object of methods you mock into var and assert on called with arguments and more)

Reply:
Good idea!